src/arrayMethodSort.js

```diff
@@ -4,8 +4,36 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  [].__proto__.sort2 = function (compareFunction) {
+    const compareFn =
```

Comment:
There is a missing assignment operator here. You should assign `compareFunction` to `compareFn` like this: `const compareFn = compareFunction || function(a, b) {...}`.